subsys/bluetooth/mesh/net.c

```diff
@@ -501,7 +501,7 @@ int bt_mesh_net_send(struct bt_mesh_net_tx *tx, struct net_buf *buf,
 
 	/* Deliver to local network interface if necessary */
 	if (bt_mesh_fixed_group_match(tx->ctx->addr) ||
-	    bt_mesh_elem_find(tx->ctx->addr)) {
+	    bt_mesh_has_addr(tx->ctx->addr)) {
 		err = loopback(tx, buf->data, buf->len);
```

Comment:
Is it necessary to loopback every message?

Reply:
Actually, I don't think this is necessary. I will add logic to bt_mesh_has_addr to avoid always returning true when the mesh message callback is in use.